tests/gendiffcov/errs/msgtest.sh

```
@@ -0,0 +1,967 @@
+    echo "ERROR: missing config file message"
+    if [ 0 == $KEEP_GOING ] ; then
+        exit 1
+    fi
+fi
+
+# read a config file which is there...
+echo "message_log = message_file.log" > testing.rc
+echo "config_file = testing.rc" > readThis.rc
+echo lcov $LCOV_OPTS --summary initial.info --config-file readThis.rc
+$COVER $LCOV_TOOL $LCOV_OPTS --summary initial.info --config-file readThis.rc
+if [ ! == ${PIPESTATUS[0]} ] ; then
```

Shellcheck (0.11.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)